src/InputGroup/index.js.flow

```diff
@@ -9,7 +9,7 @@ export type Props = {|
   ...Globals,
   ...spaceAfter,
   +label?: Translation,
-  +flex?: string | Array<string>,
+  +flex?: any | string | Array<string>, // TODO: Remove any
```

Comment:
🙏

Reply:
This was just hotfix for staging, but honestly I'm stuck on this flow error.

I just can't figure this out, in my mind this should work without problem, any idea?
```
array type [1] is incompatible with string [2].

        src/InputGroup/index.js
        167│ const InputGroup = ({
        168│   children,
        169│   label,
        170│   flex = "0 1 auto",
        171│   size = SIZE_OPTIONS.NORMAL,
        172│   help,
        173│   error,

        src/InputGroup/index.js.flow
 [2][1]  12│   +flex?: string | Array<string>,
```

Reply:
Probably the same issue 🙂 
https://github.com/kiwicom/orbit-components/pull/1345/files#diff-7e2862fad7b0396a7ff1f7216f59a81dR69-R72

It can be fixed with not assigning in the destructuration with type check.

Reply:
Perfect, thanks!